src/test/shell/bazel/bazel_workspaces_test.sh

```diff
@@ -292,6 +292,75 @@ function test_file() {
   ensure_contains_exactly 'executable: true' 1
 }
 
+function test_file_nonascii() {
```

Comment:
What are we testing? Should this be named `test_read_file_nonascii`?

Reply:
The pre-existing test `test_file()` verifies the behavior of `repository_ctx.file()`. This test verifies the behavior of `repository_ctx.file()` when the file content contains codepoints outside the ASCII range.

Reply:
Oh, fair enough. That was my lack of domain knowledge in this area showing through.